avro_schema/frontend.lua

```diff
@@ -630,6 +645,15 @@ local function create_schema(schema, options)
         end
     end
     postprocess_copy_schema(context)
+    for _, pp in ipairs(context.post_validate_default_value) do
+        local ok, res = copy_field_default(pp.field.type, pp.default)
+        if not ok then
+            local msg = string.format('Default value not valid (%s)', res)
+            li_error(pp.li, msg)
+        else
+            pp.field.default = res
```

Comment:
What if default value in a record and a field of that record have a default itself? And that default value does not set in the default for the record.

Reply:
It occurred not as easy as I thought to insert a child default to the parent.
As the "perfect behavior" is a matter of conversation, let it just work as it does.

Reply:
How does it work now? I think we should document such knowlegde. Can you update README or open an issue?

Reply:
Alex opened #103.